core/src/Cuda/Kokkos_Cuda_Parallel_Range.hpp

```diff
@@ -81,16 +81,14 @@ class ParallelFor<FunctorType, Kokkos::RangePolicy<Traits...>, Kokkos::Cuda> {
   ParallelFor& operator=(const ParallelFor&) = delete;
 
   template <class TagType>
-  inline __device__
-      typename std::enable_if<std::is_same<TagType, void>::value>::type
-      exec_range(const Member i) const {
+  inline __device__ typename std::enable_if<std::is_void<TagType>::value>::type
```

Comment:
I concur with both of the above comments.

Reply:
To be clear, I was suggesting applying Dong-Hun's further cleanup later, and just going ahead with this now

Reply:
I'd rather the `enable_if_t` cleanup be done as part of this (but it isn't a strong preference).  This is already touching 40% of the files where such a change should be made (which can also be an argument for doing that in a separate PR).

Reply:
OK to (optionally) fix the lines that are being touched.  I strongly oppose to a global `enable_if_t` refactoring as part of this PR.
Note that someone (you?) need to take this PR over and apply the changes if you really want the fix on the lines that are being edited.  Daniel is away for a little while.